docs/troubleshooting/installation.md

```diff
@@ -25,13 +25,73 @@ default via 10.10.0.10 dev harvester-mgmt proto dhcp        <-- Does a default r
 
 - Check that your DHCP server offers a default route option. Attaching content from `/run/cos/target/rke2.log` is helpful too.
 
+## Modifying cluster token on agent nodes
+
+When an agent node fails to join the cluster, it can be related to the cluster token not being identical to the server node token.
+In order to confirm the issue, connect to your agent node (i.e. with [SSH](../os#how-to-log-into-a-harvester-node)) and check the `rancherd` service log with the following command:
+
+```bash
+$ sudo journalctl -b -u rancherd
+```
+
+If the cluster token setup in the agent node is not matching the server node token, you will find several entries of the following message:
+
+```
+...
+msg="Bootstrapping Rancher (master-head/v1.21.5+rke2r1)"
+msg="failed to bootstrap system, will retry: generating plan: insecure cacerts download from https://192.168.122.115:8443/cacerts: Get \"https://192.168.122.115:8443/cacerts\": EOF"
+...
+```
+
+To fix the issue, you need to update the token value in the `rancherd` configuration file `/etc/rancher/rancherd/config.yaml`.
+
+For example, if the cluster token setup in the server node is `ThisIsTheCorrectOne`, you will update the token value as follow:
+
+```yaml
+...
+token: 'ThisIsTheCorrectOne'
+...
+```
+
+To ensure the change is persistent across reboots, update the `token` value of the OS configuration file `/oem/99_custom.yaml`:
+
+```yaml
+name: Harvester Configuration
+stages:
+  ...
+  initramfs:
+  - commands:
+    - rm -f /etc/sysconfig/network/ifroute-harvester-mgmt
+    files:
+    - path: /etc/rancher/rancherd/config.yaml
+      permissions: 384
+      owner: 0
+      group: 0
+      content: |
+        role: cluster-init
+        token: 'ThisIsTheCorrectOne' <- Update this value
```

Comment:
```suggestion
        token: 'ThisIsTheCorrectOne' # <- Update this value
```

Reply:
set the indication as a comment, just in case someone copy pasts the whole code and forgets to delete the message.